src/main/java/com/microsoft/sqlserver/jdbc/Util.java

```diff
@@ -487,9 +487,9 @@ else if (ch == ':')
                 name = SQLServerDriver.getNormalizedPropertyName(name, logger);
                 if (null != name) {
                     if (logger.isLoggable(Level.FINE)) {
-                        if ((false == name.equals(SQLServerDriverStringProperty.USER.toString()))
-                                && (false == name.equals(SQLServerDriverStringProperty.PASSWORD.toString()))
-                                && (false == name.equals(SQLServerDriverStringProperty.KEY_STORE_SECRET.toString())))
+                        if ((!name.equals(SQLServerDriverStringProperty.USER.toString()))
```

Comment:
Unnecessary brackets around `!name.equals(SQLServerDriverStringProperty.USER.toString()`, `!name.equals(SQLServerDriverStringProperty.PASSWORD.toString()`, and `!name.equals(SQLServerDriverStringProperty.KEY_STORE_SECRET.toString()`